src/gba/sio/lockstep.c

```diff
@@ -223,30 +279,30 @@ static int32_t _masterUpdate(struct GBASIOLockstepNode* node) {
 		node->p->multiRecv[2] = 0xFFFF;
 		node->p->multiRecv[3] = 0xFFFF;
 		needsToWait = true;
-		ATOMIC_STORE(node->p->d.transferActive, TRANSFER_STARTED);
-		node->nextEvent += 512;
+		SAFE_STORE(node->p->d.transferActive, TRANSFER_STARTED);
+		node->nextEvent += LOCKSTEP_INCREMENT;
```

Comment:
This was a separate increment for a reason--it's related to the minimum transfer duration.